memcached/memcached.spec

```diff
@@ -58,9 +58,11 @@ Source3:                  %{name}.service
 
 BuildRoot:                %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 
-BuildRequires:            gcc automake
+BuildRequires:            gcc automake git which
```

Comment:
Why build now require `git` and `which`? Something changed in build process?

Reply:
`git` is not required actually, but `which` is. Have a look at `autogen.sh` script:
```
# Try to locate a program by using which, and verify that the file is an
# executable
locate_binary() {
  for f in $@
  do
    file=`which $f 2>/dev/null | grep -v '^no '`
    [...]
  done
  [...]
}
```

Clean CentOS does not have `which` utility installed. So, I have added it to `BuildRequires` list.

Reply:
> Clean CentOS
Do you mean Minimal?

Reply:
Yes, minimal CentOS does not have `which` installed by default.